Khan/perseus
# Rename start and end points to point1 and point2 in Interactive Graph locked lines

## Summary:
During playtesting, it was pointed out that "start point" and "end point" are
confusing names for the defining points of a line.

Instead, we're changing them to "point1" and "point2" (which is also in line
with how Mafs names them).

Issue: https://khanacademy.atlassian.net/browse/LEMS-1989

## Test plan:
`yarn jest`

Storybook
- http://localhost:6006/?path=/story/perseuseditor-editorpage--mafs-with-locked-figures
- Open the line accordion
- Confirm that the point accordions say "point 1" and "point 2" in the summaries.

<img width="393" alt="image" src="https://github.com/Khan/perseus/assets/13231763/a8b9667c-2322-43b0-b6ff-c8a3c426eea5">
